src/Sylius/Bundle/ResourceBundle/Controller/RequestConfiguration.php

```diff
@@ -268,7 +269,12 @@ public function getLimit()
      */
     public function isPaginated()
     {
-        return (bool) $this->parameters->get('paginate', true);
+        $paginate = (bool) $this->parameters->get('paginate', true);
+        $requestPaginate = $this->request->get('paginate', $paginate);
+        if(is_string($requestPaginate)){
+            $requestPaginate = filter_var($requestPaginate, FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE);
+        }
+        return (bool) $requestPaginate;
```

Comment:
@pamil because `$requestPaginate` has more priority from `$paginate`. At line 273, `$paginate` is a default value from request, when added at query string it will have more priority from default configuration.